vtex-apps/store-components
# Fix SVG warnings

#### What is the purpose of this pull request?

Replace the unit in which the width and height of content loaders were defined.

#### What problem is this solving?

The height and width props of the [react-content-loader](https://github.com/danilowoz/react-content-loader) were being defined using percentage but as these values are used to define the viewBox of the content loader warnings were shown. This PR replace these percentage values with user units, adequate to each content loader. 

#### How should this be manually tested?
[Access this workspace](https://svgerrorpd--storecomponents.myvtex.com/iphone-xs/p), open the console. Select slow 3g in the network tab so you can see the content loaders.

#### Types of changes
